warriors/info/task_description.html

```diff
@@ -17,8 +17,8 @@
 I'm sure that many of you have some experience with computer games. But have you ever wanted to change the game so that the characters or a game world would be more consistent with your idea of the perfect game? Probably, yes.<br>
 In this mission (and in several subsequent ones, related to it) you’ll have a chance "to sit in the developer's chair" and create the logic of a simple game about battles.<br>
 Let's start with the simple task - one-on-one duel. You need to create the class <strong>Warrior</strong>, the instances of which will have 2 parameters - health (equal to 50 points) and attack (equal to 5 points), and 1 property - is_alive, which can be True (if warrior's health is > 0) or False (in the other case). In addition you have to create the second unit type - Knight, which should be the subclass of the Warrior but have the increased attack - 7. Also you have to create a function <strong>fight()</strong>, which will initiate the duel between 2 warriors and define the strongest of them. The duel occurs according to the following principle:<br>
-every turn one of the warriors will hit another one and the last will lose his health in the same value as the attack of the first warrior. After that, the second warrior will do the same to the first one.<br>
-If in the end of the turn the first warrior has > 0 health and the other one doesn’t, the function should return <strong>True</strong>, in the other case it should return <strong>False</strong>.
+Every turn, the first warrior will hit the second and this second will lose his health in the same value as the attack of the first warrior. After that, if he is still alive, the second warrior will do the same to the first one.<br>
+The fight ends with the death of one of them. If the first warrior is still alive (and the other one is not), the function should return <strong>True</strong>, and <strong>False</strong> otherwise.
```

Comment:
The fight ends with the death of one of them. If the first warrior is still alive (and thus the other one is not anymore), the function should return <strong>True</strong>, <strong>False</strong> otherwise.

Reply:
It's about what's there is in parenthesis and not about True/False, right? True/False are bold in your message, it's confusing.

Reply:
Sorry, I changed format. It is more clear?